docs/dev/table/functions.md

```diff
@@ -0,0 +1,5131 @@
+---
+title: "Built-In Functions"
+nav-parent_id: tableapi
+nav-pos: 45
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Flink Table API & SQL provides users with a set of built-in functions for data transformations. This page gives a brief overview of them.
+If a function that you need is not supported yet, you can implement a <a href="udfs.html">user-defined function</a>.
+Or if you think the function is general enough, please <a href="https://issues.apache.org/jira/secure/CreateIssue!default.jspa">open a JIRA issue</a> for it.
```

Comment:
Remove `Or` -> `If you think that the function is general enough, please open a Jira issue for it with a detailed description.`